openapi.yaml

```diff
@@ -960,3 +997,109 @@ components:
         approvingTeamDetails:
           type: string
           example: "DevRel"
+    SchemaRequest:
+      title: SchemaRequest
+      required:
+        - schemafull
+      type: object
+      properties:
+        req_no:
+          title: req_no
+          type: integer
+          format: int32
+        topicname:
+          title: Topic name
+          description: Kafka Topic name
+          type: string
+          example: "testtopic"
+        environment:
+          title: environment
+          description: Id of the environment
+          type: string
+          example: "3"
+        environmentName:
+          title: environmentName
+          description: Name of the environment
+          type: string
+          example: "DEV"
+        schemaversion:
+          title: schemaversion
+          description: SchemaRequest version
+          type: string
+          example: "1.0"
+        teamname:
+          title: Team name
+          description: Topic owner team name
+          type: string
+          example: "Infra"
+        teamId:
+          title: Team ID
+          description: Team identifier
+          type: integer
+          format: int32
+          example: 1010
+        appname:
+          title: App name
+          example: "App"
+          type: string
+        schemafull:
+          title: schemafull
+          maxLength: 2147483647
+          minLength: 8
+          type: string
+          example: "{\n  \"doc\": \"exampleTopic\",\n  \"fields\": [\n    {\n      \"default\": \"123456\",\n       \"doc\": \"test\",\n      \"name\": \"test\",\n      \"namespace\": \"test\",\n      \"type\": \"string\"\n }\n  ],\n  \"name\": \"exampleTopic\",\n  \"namespace\": \"exampleTopic\",\n  \"type\": \"record\"\n}"
+        username:
+          title: username
+          description: Username
+          example: jon.snow@klaw-project.io
+          type: string
+        requesttime:
+          title: Request time
+          type: string
+          format: date-time
+          example: 2018-11-13T20:20:39+00:00
+        requesttimestring:
+          title: Request time string representation
+          type: string
+          example: "28-Dec-2022 14:54:57"
+        topicstatus:
+          title: Topic status
+          type: string
+          example: "created"
+        requesttype:
+          title: Type of request
+          type: string
+          example: "Create"
+        remarks:
+          title: Remarks
+          description: SchemaRequest specific comment
+          pattern: "^$|^[a-zA-Z 0-9_.-]{3,}$"
+          type: string
+          example: "Please approve"
+        approver:
```

Comment:
Probably nice to have an example for this, I assume it's username but might be email?

Reply:
I don't know 🙈  I checked in the app the real responses, but there were only responses with `null`. This leads to another question - when do we add `nullable` 🤔

Reply:
I talked with Murali and I think I remembered he thought that no field should have `null` value, it should simply not be returned 🤔 But probably better to ask confirmation.

Reply:
@muralibasani  - would `approver` be an email? Then I would add that as an example.

Reply:
approver is login username which can be email or username